assets/js/signup.js

```diff
@@ -23,7 +11,7 @@ var loadQuestions = function() {
   });
 
   $.getJSON({
-    url: '/server/form.php?retrieve=questions',
+    url: '/server/form.php?retrieve=questions&subheadings[]=Contact Information&subheadings[]=Language Information&subheadings[]=Background Information',
```

Comment:
Understood, I could add an index to the questions and also a subheading index which specifies the ordering of questions and of subheadings, respectively.